docs/guide/provider-migration.md

```diff
@@ -43,6 +43,8 @@ We made the following breaking changes to the `window.ethereum` API:
     Its removal _may_ affect those who do not use it directly, e.g. if another library you use relies on the object.
 * Remove the `ethereum.autoRefreshOnNetworkChange` property
   * Consumers can still set this property on the provider, it just won't do anything.
+* Removed the `web3.currentProvider` method
```

Comment:
We still support `web3.currentProvider`, but it is deprecated and emits a warning when used.

```suggestion
* Deprecate the `web3.currentProvider` method
```